transform.go

```diff
@@ -64,7 +65,7 @@ func Transform(img []byte, opt Options) ([]byte, error) {
 		if err != nil {
 			return nil, err
 		}
-	case "jpeg", "webp": // default to encoding webp as jpeg
+	case "jpeg", "webp", "tiff": // default to encoding webp & tiff as jpeg
```

Comment:
why can't we encode as tiff as well using https://godoc.org/golang.org/x/image/tiff#Encode ?

Reply:
I suppose we probably can. I was trying to make sure the file would be visible "in-browser" and generally tiff is not supported so it needs to be either jpg or png for that.

Reply:
Ah, good call.  I didn't realize how poorly supported tiff was in browsers :)